pkgs/top-level/all-packages.nix

```diff
@@ -266,6 +266,8 @@ with pkgs;
 
   astrolog = callPackage ../applications/science/astronomy/astrolog { };
 
+  astrometry.net = callPackage ../applications/science/astronomy/astrometry.net { };
```

Comment:
Outside of `pname`, the package name shouldn't include a dot, but instead something like `astrometry-net`. Including a dot here makes it equivalent to
```
astrometry = {
  net = callPackage ../applications/science/astronomy/astrometry.net { };
};
```
as a attrset with a single key.

An example of this elsewhere is [`wheezy.captcha`](https://github.com/NixOS/nixpkgs/blob/59713b5555cc182545f81422c596089e871c9103/pkgs/development/python-modules/wheezy-captcha/default.nix), which gets renamed to `wheezy-captcha`.

Reply:
Done!